src/signals/incident/definitions/components/index.js

```diff
@@ -0,0 +1,9 @@
+import A from './A';
+import Concat from './Concat/';
+import Ul from './Ul/';
+
+export default {
+  A,
+  Concat,
+  Ul
+};
```

Comment:
Hmm... Quite a lot of work to reintroduce normal HTML components for this generator syntax...